SECURITY.md

```diff
@@ -1,6 +1,6 @@
-# Security within Superface SDK
+# Security within Superface OneSDK
 
-The Superface SDK interprets the map to fulfill the required usecase. This often requires making http requests with authorization. Thus the user needs to provide Superface with the right secrets to access the capability exposed by the API.
+OneSDK interprets the map to fulfill the required usecase. This often requires making http requests with authorization. Thus the user needs to provide Superface with the right secrets to access the capability exposed by the API.
```

Comment:
I do not understand what this is saying and actually it is raising some concerns in my eyes... maybe rephrase it?